### ISSUE TITLE
Go get error -github.com/extrame/xls/comparexlsxlsx.go:27:38: xlsxSheet.Rows undefined (type *xlsx.Sheet has no field or method Rows)

### ISSUE THREAD

**Issue by pphowakande** [extrame/xls#71]
when I try to import this into my GOPATH, using command -
go get github.com/extrame/xls
I get error as -

# github.com/extrame/xls
github.com/extrame/xls/comparexlsxlsx.go:27:38: xlsxSheet.Rows undefined (type *xlsx.Sheet has no field or method Rows)

any one has an idea why am I getting this error ?

**Comment by faxiang1230** [comment]
Yeah, I also meet the issue,maybe `xlsx` repo update and conflict
`# github.com/extrame/xls
/home/linux/go/src/github.com/extrame/xls/comparexlsxlsx.go:27:38: xlsxSheet.Rows undefined (type *xlsx.Sheet has no field or method Rows)
`

**Comment by sergeilem** [comment]
See PR #65
